### PR TITLE
Breaking

### PR DESCRIPTION
**Pull request by rowanmanning**
Opening this because it’s a nicer review experience :)

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-cookie-message bundle size difference from 4.7.3 to 2eba212a1bf8f4c7e3eda7699cde6d4cc0d06043
js: 17.45kb decrease (-4.54kb/gzip)
css: 5.1kb decrease (-0.44kb/gzip)

File: README.md [hunk 1]
```diff
@@ -98,61 +106,6 @@ If you need a different message, this can be added with HTML of your choosing. F
...
-```html
-<div data-o-component="o-cookie-message" class='o-cookie-message o-cookie-message--banner-centric'>
-</div>
-```
-
-If you do not want to use the FT legal team approved cookie message (for example if you aren't running an FT site), you can opt out of it by setting the `data-o-cookie-message-use-custom-html` attribute to `"true"`. All FT sites **must** use this cookie message unless otherwise cleared by the FT's legal team.
-
----
-
 ## Contact
```
Comment (rowanmanning) [inline README.md]:
```suggestion
## Migration

State | Major Version | Last Minor Release | Migration guide |
:---: | :---: | :---: | :---:
✨ active | 5 | N/A | [migrate to v5](MIGRATION.md#migrating-from-v4-to-v5) |
⚠ maintained | 4 | 4.7 | [migrate to v4](MIGRATION.md#migrating-from-v3-to-v4) |
╳ deprecated | 3 | 3.3 | [migrate to v3](MIGRATION.md#migrating-from-v2-to-v3) |
╳ deprecated | 2 | 2.2 | [migrate to v2](MIGRATION.md#migrating-from-v1-to-v2) |
╳ deprecated | 1 | 1.2 | - |

## Contact
```

Comment (rowanmanning) [reply README.md]: I just saw that you committed some README stuff just before this, sorry :) feel free to ignore if you were doing anyway

**Comment by origamiserviceuser** [comment]
o-cookie-message bundle size difference from 4.7.3 to 7ccb7835b6a3c551771075ed760f3c4ea1d3fe9c
js: 17.45kb decrease (-4.54kb/gzip)
css: 5.1kb decrease (-0.44kb/gzip)

**Comment by origamiserviceuser** [comment]
o-cookie-message bundle size difference from 4.7.3 to 04b1b844af70846e4e3a5689a3daa2ab056653d2
js: 17.45kb decrease (-4.54kb/gzip)
css: 5.1kb decrease (-0.44kb/gzip)

**Comment by origamiserviceuser** [comment]
o-cookie-message bundle size difference from 4.7.3 to bcd7f42d91c0791e855bd90f2a0683a8d9a8dc9a
js: 17.45kb decrease (-4.54kb/gzip)
css: 5.1kb decrease (-0.44kb/gzip)

**Pull request merged**